src/modules/auth/login.controller.ts

```diff
@@ -20,9 +20,9 @@ import { RoleService } from '../role/role.service';
 @ApiTags('Auth')
 @Controller({ version: '1' })
 export class LoginController {
+  private cookiesServices = new CookiesServices();
```

Comment:
I think this is unlikely to change snyk's results and we lose the benefits of nest Dependency Injection. I think that I would try copying the `CookiesService` code directly into the LoginController